app/uk/gov/hmrc/economiccrimelevyregistration/models/EntityType.scala

```diff
@@ -30,6 +30,7 @@ case object ScottishPartnership extends EntityType
 case object LimitedPartnership extends EntityType
 case object ScottishLimitedPartnership extends EntityType
 case object LimitedLiabilityPartnership extends EntityType
+case object OtherEntityType extends EntityType
```

Comment:
Any particular reason we can't just call this `Other`?

Reply:
I did originally call it 'Other' but it failed to compile saying there was already something with the same name. I didn't bother investigating what this was (and I can't remember exactly what the conflict was), I just renamed this. Do you want me to investigate further or leave it as-is?

Reply:
I think it probably was complaining because it clashes with the `Other` AML supervisor type, but that should be fine as it's qualified by the package as long as we import it correctly in other places?

Either way we should probably be consistent, so either `Other` throughout or rename the AML supervisor `Other` type to `OtherAmlSupervisorType`